tests/x11/yast2_snapper.pm

```diff
@@ -20,7 +20,7 @@
 # - Close yast2 snapper, delete testadata
 # Maintainer: Richard Brown <rbrown@suse.de>
 
-use base qw(y2snapper_common x11test);
+use base qw(opensusebasetest y2snapper_common x11test);
```

Comment:
you should not need to add `opensusebasetest` as the `x11test.pm` inherits from it already. And it should work just as it is the multi inheritance also should be avoided. inheriting from three classes could be a flag in the design. Do you have any reason for it?

Reply:
Done